datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/codeowners.py

```diff
@@ -59,6 +81,10 @@ def codeowners():
 
     has_failed = False
     codeowner_map = create_codeowners_map()
+    owners_resolver = create_codeowners_resolver()
+    if not validate_logs_assets_codeowners():
+        echo_success("All integrations have valid logs codeowners.")
```

Comment:
Note that before making the validation fail for this check I will have to update the codeowners file.

Reply:
We'll add some automatic tests for this later.